packages/now-static-build/build.sh

```diff
@@ -3,8 +3,9 @@ set -euo pipefail
 
 ncc build src/index.ts -o dist
 
-# copy gatsby plugin files in the gatsby redirect test case
+# copy gatsby-plugin-now files to the dist folder to
```

Comment:
Could we eventually move the `gatsby-plugin-now` source and tests into `./packages/now-static-build/src` and then we wouldn't need this additional build step?

Reply:
Yes! But for now I want to publish a latest version of the plugin and add the deprecation message to the README 🙂